lib/manager/gradle/build-gradle.ts

```diff
@@ -274,7 +295,7 @@ function updatePropertyFileGlobalVariables(
 ): string | null {
   const variable = variables[`${dependency.group}:${dependency.name}`];
   if (variable) {
-    const regex = new RegExp(`(${variable}\\s*=\\s*)(.*)`);
+    const regex = regEx(`(${variable}\\s*=\\s*)(.*)`);
     const match = regex.exec(buildGradleContent);
     if (match) {
       return buildGradleContent.replace(regex, `$1${newVersion}`);
```

Comment:
```suggestion
    const regex = regEx(`(${variable}\\s*=\\s*)(.*)`);
    const match = regex.exec(buildGradleContent);
    if (match) {
      return buildGradleContent.replace(match[0], `${match[1]}${newVersion}`);
```

Reply:
@Jnosh See suggestion above, should fix your problem.

Reply:
@viceice Thank you! That seems to have done the trick.